src/routes/docs/advanced/self-hosting/update/+page.markdoc

```diff
@@ -28,6 +28,10 @@ This is the parent directory where you will find the appwrite directory, inside
 
 # Installing the next version {% #install-next-version %}
 
+{% info title="Parent directory naming" %}
+If you have a different parent directory name (such as `docker` rather than `appwrite`), use the `--no-start=true` flag at the end of the `docker run` command, and then start the Docker Compose stack manually to avoid conflicts.
```

Comment:
We don't really want to delve to much into this and would really rather people stick with the standard name so it would be best to put:

```suggestion
Your Appwrite installation's parent directory name is expected to be `appwrite`. Changing the directory name will result in a miss-matched docker project names.
```